test/unit/payment_gateway_setting_test.rb

```diff
@@ -47,16 +47,6 @@ def setup
     assert @gateway_setting.configured?
   end
 
-  test '#configured? failed if any gateway_settings missing for adyen12' do
```

Comment:
Would it make sense to keep this test but treating adyen as an unknown gateway_type? To ensure nobody uses it or it isn't installed by error or something...

Reply:
To ensure that nobody uses it, I will run a rake task before merging this.
So by the time that I merge this, it will be safe.

Reply:
https://github.com/3scale/porta/pull/2300
The goal of that is to have 2 rake tasks:
1. Save in a file info of providers actively using a payment gateway setting. Meaning specifically: id. admin_domain and state of non-scheduled_for_deletion providers that have a payment gateway setting of the type of the param that is configured (usable).
2. Remove all payment gateway setting records in the DB of a specific type. Using this one without having the first one would be crazy, but if the first one is empty or product gives up the 'ok' (highly likely), then we can run this one and forget in the code that Adyen ever existed.